src/core/BUILD

```diff
@@ -2716,6 +2716,27 @@ grpc_cc_library(
     ],
 )
 
+grpc_cc_library(
+    name = "grpc_audit_logging",
+    srcs = [
+        "lib/security/audit_logging/audit_logging.cc",
```

Comment:
I think the audit_logging directory should be under the authorization directory.

Reply:
Moved them over.